src/doc/src/SUMMARY.md

```diff
@@ -9,6 +9,7 @@
 * [Cargo Guide](guide/index.md)
     * [Why Cargo Exists](guide/why-cargo-exists.md)
     * [Creating a New Package](guide/creating-a-new-project.md)
+    * [Creating a Workspace](guide/creating-a-new-workspace.md)
     * [Working on an Existing Package](guide/working-on-an-existing-project.md)
```

Comment:
Within the scope of things, we are introducing workspaces before we introduce dependencies, tests, etc.  This likely should be later